src/DotVVM.Framework/Controls/GridViewValueColumn.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Binding;
+using DotVVM.Framework.Binding.Expressions;
+using DotVVM.Framework.Hosting;
+
+namespace DotVVM.Framework.Controls
+{
+    /// <summary>
+    /// A base class for GridView columns that present a value by wrapping a control.
+    /// </summary>
+    public abstract class GridViewValueColumn : GridViewColumn
+    {
+        /// <summary>
+        /// Gets or sets a binding which retrieves the value to display from the current data item.
+        /// </summary>
+        [MarkupOptions(AllowHardCodedValue = false, Required = true)]
+        public IValueBinding ValueBinding
```

Comment:
I don't think it's a great idea to loose the type constraint of `ValueBinding` property.